script/deploy/devnet/M2_Deploy_From_Scratch.s.sol

```diff
@@ -88,6 +91,17 @@ contract Deployer_M2 is Script, Test {
     uint256 DELEGATION_WITHDRAWAL_DELAY_BLOCKS;
     uint256 EIGENPOD_MANAGER_INIT_PAUSED_STATUS;
     uint256 DELAYED_WITHDRAWAL_ROUTER_INIT_PAUSED_STATUS;
+    uint256 REWARDS_COORDINATOR_INIT_PAUSED_STATUS;
```

Comment:
move it down with the other RewardsCoordinator vars?

Reply:
moved the delayed box below